# MoveDistance.srv
# 모바일 로봇 거리 이동 서비스 정의
# 
# Request:
#   distance - 이동할 거리 (m), 양수=전진, 음수=후진
#
# Response:
#   success - 이동 성공 여부
#   message - 결과 메시지

# Request
float32 distance    # 이동할 거리 (m), 음수는 후진
---
# Response
bool success
string message

